Dockerfile-checklicence

```diff
@@ -46,7 +46,8 @@ ENV RAT_VERSION="${RAT_VERSION}" \
     RAT_URL="http://repo1.maven.org/maven2/org/apache/rat/apache-rat/${RAT_VERSION}/apache-rat-${RAT_VERSION}.jar"
 
 RUN echo "Downloading RAT from ${RAT_URL} to ${RAT_JAR}" \
-    && curl -sL ${RAT_URL} > ${RAT_JAR}
+    && curl -sL ${RAT_URL} > ${RAT_JAR}  \
+    && jar -tf ${RAT_JAR}
```

Comment:
Maybe we can do a checksum to verify the contents: https://www.apache.org/info/verification.html

Reply:
Good Idea. Added!